modules/config/home-cursor.nix

```diff
@@ -104,6 +104,12 @@ in {
         backends, you can toggle them via the enable option. For example,
         <xref linkend="opt-home.pointerCursor.x11.enable"/>
         will enable x11 cursor configurations.
+	</para><para>
+        Note that this will merely generate the cursor configurations.
+	To apply the configurations, the relevant subsytems must also be configured.
+	For example, <xref linkend="opt-home.pointerCursor.gtk.enable"/> will generate
+        the gtk cursor configuration, but <xref linkend="opt-gtk.enable"/> needs
+        to be set for it to be applied.
```

Comment:
You seem to have mixed spaces and tabs here

Reply:
oh shoot, you're right. I haven't gotten around to configure vim on this machine yet, so I didn't consider that it made tabs. In any case, It should be consistent now.